test/cartesian/ticks/getTicks.spec.ts

```diff
@@ -27,9 +28,9 @@ const EXAMPLE_INPUT = {
   y: 100,
 };
 
-jest.mock('../../src/util/DOMUtils', () => ({
+jest.mock('../../../src/util/DOMUtils', () => ({
   // We mock string size measurement, because getStringSize else returns 0 in these tests.
-  getStringSize: jest.fn(() => ({ width: 20, height: 20 })),
+  getStringSize: jest.fn((text: string) => ({ width: text.length, height: 20 })),
```

Comment:
I don't think this adds any indication of actual text length. Better maybe than no variation, but certainly not accurate.

Reply:
Agree, but this enables me to test with larger labels on different positions (i.e. first tick, second tick, third tick, etc.)